apps/meteor/client/views/account/omnichannel/PreferencesConversationTranscript.tsx

```diff
@@ -1,24 +1,36 @@
 import { Accordion, Box, Field, FieldGroup, FieldLabel, FieldRow, FieldHint, Tag, ToggleSwitch } from '@rocket.chat/fuselage';
 import { useUniqueId } from '@rocket.chat/fuselage-hooks';
-import { useTranslation, usePermission } from '@rocket.chat/ui-contexts';
-import React from 'react';
+import { useTranslation, usePermission, useSetting } from '@rocket.chat/ui-contexts';
+import React, { useEffect } from 'react';
 import { useFormContext } from 'react-hook-form';
 
 import { useHasLicenseModule } from '../../../hooks/useHasLicenseModule';
 
 const PreferencesConversationTranscript = () => {
 	const t = useTranslation();
 
-	const { register } = useFormContext();
+	const { register, setValue, watch } = useFormContext();
 
 	const hasLicense = useHasLicenseModule('livechat-enterprise');
+	const alwaysSendEmailTranscript = useSetting('Livechat_transcript_send_always');
 	const canSendTranscriptPDF = usePermission('request-pdf-transcript');
-	const canSendTranscriptEmail = usePermission('send-omnichannel-chat-transcript');
+	const canSendTranscriptEmailPermission = usePermission('send-omnichannel-chat-transcript');
+	const canSendTranscriptEmail = canSendTranscriptEmailPermission && !alwaysSendEmailTranscript;
+	const canSendTranscriptEmailPref = watch('omnichannelTranscriptEmail');
 	const cantSendTranscriptPDF = !canSendTranscriptPDF || !hasLicense;
 
 	const omnichannelTranscriptPDF = useUniqueId();
 	const omnichannelTranscriptEmail = useUniqueId();
 
+	useEffect(() => {
```

Comment:
Let's not use useEffect here, if there is a need to be reactive to setting changes, we'll have to change the whole page